js-basics/js-create-elements/js-createelement_social-media-post/js/index.js

```diff
@@ -10,3 +10,26 @@ likeButton.addEventListener("click", handleLikeButtonClick);
 
 // Exercise:
 // Use document.createElement() and append another social media post to the body.
+const postBox = document.createElement("section");
+
+const postContent = document.createElement("p");
+postContent.textContent = "Nah, i'd win.------------------";
+
+const postFooter = document.createElement("footer");
+
+const postUsername = document.createElement("span");
+postUsername.textContent = "@Gojo";
+
+const postLikeButton = document.createElement("button");
+postLikeButton.textContent = "♥ Like";
+postLikeButton.addEventListener("click", handleLikeButtonClick);
+
+postBox.classList.add("post");
+postContent.classList.add("post__content");
+postFooter.classList.add("post__footer");
+postUsername.classList.add("post__username");
+postLikeButton.classList.add("post__button");
+
+postBox.append(postContent, postFooter, postUsername, postLikeButton);
```

Comment:
did it work with only this one line?
i had to do it this way:
`postSection.append(postContent, postFooter);`
`postFooter.append(postCreator, postLikeButton);`